ogr/ogrsf_frmts/miramon/mm_wrlayr.c

```diff
@@ -0,0 +1,7038 @@
+                // Horizontal Reference System
+                fprintf_function(pF, "%s=plane" LineReturn,
+                                 KEY_HorizontalSystemIdentifier);
+                fprintf_function(pF, "%s=local" LineReturn,
+                                 KEY_HorizontalSystemDefinition);
+                if (hMMMD->pXUnit)
+                    fprintf_function(pF, "%s=%s" LineReturn, KEY_unitats,
+                                     hMMMD->pXUnit);
+                if (hMMMD->pYUnit)
+                {
+                    if (!hMMMD->pXUnit ||
+                        strcasecmp(hMMMD->pXUnit, hMMMD->pYUnit))
+                        fprintf_function(pF, "%s=%s" LineReturn, KEY_unitatsY,
+                                         hMMMD->pYUnit);
+                }
+            }
+        }
+        else
+        {
+            fprintf_function(pF, "%s=plane" LineReturn,
+                             KEY_HorizontalSystemIdentifier);
+            fprintf_function(pF, "%s=local" LineReturn,
+                             KEY_HorizontalSystemDefinition);
+            if (hMMMD->pXUnit)
+            {
+                fprintf_function(pF, "%s=%s" LineReturn, KEY_unitats,
+                                 hMMMD->pXUnit);
+                if (hMMMD->pYUnit)
+                {
+                    if (!hMMMD->pXUnit ||
+                        strcasecmp(hMMMD->pXUnit, hMMMD->pYUnit))
+                        fprintf_function(pF, "%s=%s" LineReturn, KEY_unitatsY,
+                                         hMMMD->pYUnit);
+                }
+            }
+        }
+    }
+
+    // Writing OVERVIEW:ASPECTES_TECNICS in polygon metadata file.
+    // ArcSource=fitx_pol.arc
+    if (hMMMD->ePlainLT == MM_LayerType_Pol)
+    {
+        fprintf_function(pF, LineReturn "[%s]" LineReturn,
+                         SECTION_OVVW_ASPECTES_TECNICS);
+        fprintf_function(pF, "%s=\"%s\"" LineReturn, KEY_ArcSource,
+                         hMMMD->aArcFile);
+    }
+
+    // Writing EXTENT section
+    fprintf_function(pF, LineReturn "[%s]" LineReturn, SECTION_EXTENT);
+    fprintf_function(pF, "%s=0" LineReturn, KEY_toler_env);
+
+    if (hMMMD->hBB.dfMinX != MM_UNDEFINED_STATISTICAL_VALUE &&
+        hMMMD->hBB.dfMaxX != -MM_UNDEFINED_STATISTICAL_VALUE &&
+        hMMMD->hBB.dfMinY != MM_UNDEFINED_STATISTICAL_VALUE &&
+        hMMMD->hBB.dfMaxY != -MM_UNDEFINED_STATISTICAL_VALUE)
+    {
+        fprintf_function(pF, "%s=%lf" LineReturn, KEY_MinX, hMMMD->hBB.dfMinX);
+        fprintf_function(pF, "%s=%lf" LineReturn, KEY_MaxX, hMMMD->hBB.dfMaxX);
+        fprintf_function(pF, "%s=%lf" LineReturn, KEY_MinY, hMMMD->hBB.dfMinY);
+        fprintf_function(pF, "%s=%lf" LineReturn, KEY_MaxY, hMMMD->hBB.dfMaxY);
+    }
+
+    // Writing OVERVIEW section
+    fprintf_function(pF, LineReturn "[%s]" LineReturn, SECTION_OVERVIEW);
+
+    currentTime = time(nullptr);
+    pLocalTime = localtime(&currentTime);
```

Comment:
localtime() isn't thread safe. For GDAL builds, you can use the VSILocalTime() function

Reply:
Done.